packages/api/server.js

```diff
@@ -52,7 +52,7 @@ app.delete('/api/repo/:orgname/:reponame/test/:testid', async (req, res) => {
   });
 
   const url = `http://github.com/login/oauth/authorize?client_id=${process.env.CLIENT_ID}&state=${state}&allow_signup=false&scope=repo`;
-  res.status(302).redirect(url);
+  res.status(200).send(url).end();
```

Comment:
I don't think we should need to call `.end()`.

Reply:
Also, isn't 200 the default code in express? I think `res.send('...')` sends 200 as the status code by default?